### ISSUE TITLE
Bluesky Account Only Exists Under Invalid Handle (Mastodon -> Bluesky)

### ISSUE THREAD

**Issue by NinjaCheetah** [snarfed/bridgy-fed#1581]
I recently bridged my Mastoson account to Bluesky, but the normal handle that appears on the Bridgy Fed account status page doesn’t actually exist on Bluesky. Instead, my likes and replies appear under an invalid handle.

The intended handle is [@NinjaCheetah.mastodon.online.ap.brid.gy](https://bsky.app/profile/NinjaCheetah.mastodon.online.ap.brid.gy), which doesn’t exist, and my likes and replies appear under the handle [@did:plc:arwfgynzynrhaabylycgqjy4](https://bsky.app/profile/did:plc:arwfgynzynrhaabylycgqjy4), which Bluesky has marked with “Invalid Handle”.

The bridge does appear to be working as intended otherwise, but the handle being wrong makes it hard to share.

**Comment by snarfed** [comment]
Oof, sorry for the trouble! We ended up missing the DNS record for your handle. I've added it manually, it's working now: https://bsky.app/profile/NinjaCheetah.mastodon.online.ap.brid.gy

Looking at logs, I see that you blocked @bsky.brid.gy@bsky.brid.gy to disable the bridge three times, and followed it to enable the bridge eight times, between 2024-11-26 21:45:05 PT and 2024-11-29 15:48:39 PT. Totally ok, Bridgy Fed should be able to handle that, but some of them were right after one another, I suspect something like #1338 may have caused this. Sorry!
